### PR TITLE
Updated to allow for specifying the root volume via SDB

### PR DESCRIPTION
**Pull request by blarghmatey**

### CODE REVIEW

File: salt/orchestrate/aws/cloud_profiles/backup_host.conf [hunk 1]
```diff
@@ -14,7 +14,7 @@ backup_host:
     roles:
       - backups
   block_device_mappings:
-    - DeviceName: /dev/xvda
+    - DeviceName: {{ salt.sdb.get('sdb://consul/debian_root_device')|default('/dev/xvda') }}
```
Comment (shaidar) [inline salt/orchestrate/aws/cloud_profiles/backup_host.conf]: Isn't the ubuntu root device the same name?

Comment (blarghmatey) [inline salt/orchestrate/aws/cloud_profiles/backup_host.conf]: Actually, the newer ubuntu devices use the /dev/sda1 device name...

Comment (blarghmatey) [inline salt/orchestrate/aws/cloud_profiles/backup_host.conf]: Also, the backup host uses Debian and not Ubuntu

**Comment by shaidar** [comment]
:thumbsup:

**Pull request merged**